armi/tests/test_plugins.py

```diff
@@ -92,6 +105,15 @@ def test_defineFlags(self):
         # show the flag exists now
         self.assertEqual(type(Flags.SUPER_FLAG._value), int)
 
+    def test_axialExpansionHook(self):
+        """Test that plugins can override the axial expansion of assemblies via a hook."""
+        pm = self.app.pluginManager
+        first = pm.hook.getAxialExpansionChanger()
+        self.assertIs(first, AxialExpansionChanger)
+        pm.register(SillyAxialPlugin)
```

Comment:
same posterity comment here: can you add an inline comment at line 113 saying something like "now register the Plugin (ARMI App) with the subclass. the axial expansion changer class pulled from the plugin system is the `SillyAxialExpansionChanger` instead of the default in ARMI"

Reply:
Sure thing!

Reply:
b1270c93